src/frontend/web_application/src/components/Dropdown/style.scss

```diff
@@ -4,7 +4,13 @@
 .m-dropdown {
   @include dropdown-container;
   width: auto;
-
   background-color: $co-color__fg__back;
   box-shadow: $co-shadow;
+
+  &--flat {
```

Comment:
ok the problem comes from predefined styles by foundation. I had the same pbm with ApplicationSwitcher. 

So you only need to use the `Raw` component instead of Dropdown then make your own styles